src/services.js

```diff
@@ -342,6 +342,68 @@ export class Services {
     return getService(win, 'localization');
   }
 
+  /**
+   * TODO(#14357): Remove this when amp-story:0.1 is deprecated.
+   * @param {!Window} win
+   * @return {?Promise<?../extensions/amp-story/0.1/variable-service.StoryVariableDef>}
+   */
+  static storyVariableServiceForOrNullV01(win) {
```

Comment:
Can we reuse the same services, passing in an optional parameter for the version?

Reply:
This will be pretty complicated since the services' functions return different types, so we'll have to change every usage downstream